ouroboros-consensus-cardano/src/ouroboros-consensus-cardano/Ouroboros/Consensus/Cardano/CanHardFork.hs

```diff
@@ -203,6 +210,14 @@ instance CardanoHardForkConstraints c => CanHardFork (CardanoEras c) where
                   (translateTxBabbageToConwayWrapper ctxt)
                   (translateValidatedTxBabbageToConwayWrapper ctxt)
         )
+      $ PCons
+        ( RequireBoth $ \_cfgConway cfgDijkstra ->
+            -- TODO(geo2a): do we need cfgConway? Seems like we do not.
```

Comment:
Yes, we don't need it, but we currently don't have a variant of `Require*` that gives us just the second argument (we have `Requiring` which gives us the first one). Seems fine to keep it like this.

Reply:
Do we want to create an issue for adding said variant?

Reply:
I have removed the TODO, and I think it's OK to follow the existing pattern here.